### PR TITLE
Update the Android browser to the 6.0 version

### PR DESCRIPTION
**Pull request by LavrovArtem**

### CODE REVIEW

**Comment by LavrovArtem** [comment]
@testcafe-build-bot \retest

**Comment by LavrovArtem** [comment]
@testcafe-build-bot \retest

File: test/client/fixtures/sandbox/fetch-test.js [hunk 1]
```diff
@@ -429,10 +429,10 @@ if (window.fetch) {
             test('nested Requests', function () {
                 var request1       = new Request('/echo-request-headers', initWithHeader1);
                 var request2       = new Request(request1);
-                var request3       = new Request(request1, initWithHeader3);
+                var request3       = new Request(request2, initWithHeader3);
```
Comment (miherlosev) [inline test/client/fixtures/sandbox/fetch-test.js]: Revert changes for this test

**Comment by miherlosev** [comment]
Also try use `Object.getOwnPropertyNames` for https://github.com/DevExpress/testcafe-hammerhead/blob/6b3dc44526bb070685ea5f378f06c83384dec82c/src/client/sandbox/storages/wrapper.js#L41
and add a test if it is necessary

File: src/client/sandbox/node/window.js [hunk 2]
```diff
@@ -320,8 +320,12 @@ export default class WindowSandbox extends SandboxBase {
                 const url = args[0];
 
                 if (typeof url === 'string') {
-                    if (WindowSandbox._isSecureOrigin(url))
-                        return Promise.reject(new DOMException('Only secure origins are allowed.', 'SecurityError'));
+                    if (WindowSandbox._isSecureOrigin(url)) {
+                        // NOTE: the android browser cannot create the DOMException error manually
```
[the remark this answers is not in the document]
Comment (LavrovArtem) [reply src/client/sandbox/node/window.js]: My version:
// NOTE: We cannot create an instance of the DOMException in the Android 6.0 browser.
// The 'TypeError: Illegal constructor' error is raised if we try to call the constructor.

[the remark this answers is not in the document]
Comment (miherlosev) [reply src/client/sandbox/node/window.js]: Ok. Let it be

**Comment by LavrovArtem** [comment]
@testcafe-build-bot \retest

**Comment by LavrovArtem** [comment]
@testcafe-build-bot \retest

**Comment by miherlosev** [comment]
rebase please

**Comment by LavrovArtem** [comment]
@testcafe-build-bot \retest

File: test/client/fixtures/sandbox/fetch-test.js [hunk 3]
```diff
@@ -410,6 +411,12 @@ if (window.fetch) {
         });
 
         module('should emulate native behaviour on headers overwriting', function () {
+            // NOTE: These tests fail in the Android 6.0 browser with the following errors:
+            // Failed to construct 'Request': Cannot construct a Request with a Request object that has already been used.
+            // Failed to execute 'fetch' on 'Window': Cannot construct a Request with a Request object that has already been used.
+            if (browserUtils.isAndroid)
```
Comment (miherlosev) [inline test/client/fixtures/sandbox/fetch-test.js]: We usually use browser-related conditions before test.
So, change condition and comment
```js
// NOTE: In Android 6.0 it is impossible to initialize Request using another Request as parameter.
// https://developer.mozilla.org/en-US/docs/Web/API/Request/Request
// https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/fetch
if (!browserUtils.isAndroid) {
    module('should emulate native behaviour on headers overwriting'
}
```

**Comment by LavrovArtem** [comment]
ping @churkin

**Pull request merged**